Sofa/framework/Helper/src/sofa/helper/LCPcalc.cpp

```diff
@@ -220,13 +220,18 @@ void LCP::solveNLCP(bool convergenceTest, std::vector<SReal>* residuals, std::ve
 }
 
 
+int resoudreLCP(int dim, SReal * q, SReal ** M, SReal * res)
+{
+    return solveLCP(dim, q, M, res);
+}
+
 //#include "mex.h"
 /* Resoud un LCP écrit sous la forme U = q + M.F
  * dim : dimension du pb
  * res[0..dim-1] = U
  * res[dim..2*dim-1] = F
  */
-int resoudreLCP(int dim, SReal * q, SReal ** M, SReal * res)
+int solveLCP(int dim, SReal * q, SReal ** M, SReal * res)
 {
 
     // déclaration des variables
```

Comment:
Lots of French in here  (comments, variable names...) 😶‍🌫️

Reply:
Yes, it needs to be change, but I wanted to focus only on the function names in this PR